### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by tomelgato** [davetcc/tcMenu#311] [General]
Is there anywhere a summary which changes/new features happened between the major releases (2 -> 2.5 -> 3.0)? The linked docs are quite outdated.

**Comment by davetcc** [comment]
Do you have any specific areas where the documentation is missing? It's not great, but it is more or less up to date at:

https://www.thecoderscorner.com/products/arduino-libraries/tc-menu/

The best place to look at the new features added in each release is here on GitHub, in the release notes for each tcMenu release, and at https://github.com/davetcc/tcMenuLib/releases for the library releases. Likewise, you can see any new features in the other dependent libraries in the same way.

As part of 3.2 the documentation will be open-sourced so that others can make corrections and improve.

**Reply by tomelgato** [reply]
Thanks Dave! I take another look :)
